test/UnitTests/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "compilationOptions": {
+    "warningsAsErrors": "true"
+  },
+  "dependencies": {
+    "MusicStore": { "type": "build", "version": "1.0.0-*" },
+    "Microsoft.AspNet.Mvc" : "6.0.0-*",
+    "xunit.runner.kre": "1.0.0-*"
+  },
+  "commands": {
+    "test": "xunit.runner.kre"
+  },
+  "frameworks": {
+    "aspnet50": { }
```

Comment:
Sorry, I didn't pay attention when I copied this from an MVC unit test.. I updated this.